loader/CMakeLists.txt

```diff
@@ -279,10 +279,6 @@ if(WIN32)
 
     target_link_libraries(vulkan PRIVATE loader_specific_options loader-opt)
 
-    if (UPDATE_DEPS)
-        add_dependencies(vulkan vl_update_deps)
-    endif()
```

Comment:
This target no longer exists.

Reply:
I probably added that in a bodged attempt to fix an issue where update_deps wasn't running when I expected it to. No problem removing that.